Update flowetl.Dockerfile (#6685)
Flowminder/FlowKit: CHANGELOG.md

- FlowETL now attached new data as partitions, rather than subtables
> [!WARNING]
> This change is not backwards compatible with earlier releases of FlowDB, and you will need to repopulate your deployment. We recommend combining this change with the new parquet support.

> - FlowETL airflow bumped to 2.9.2 p

Reviewer: ```suggestion
- FlowETL is now built on Airflow 2.9.2
```
